
number_one = 5
number_two = '2'

try:
    print(number_one + number_two)

except:
    print('Se ha producido un error')

else: # Opcional
    # Se ejecuta si no hay un error
    print('La ejecución continúa')

finally: # Opcional
        # Se ejecuta siempre
        print('La ejecución continúaaaaaa')
